.github/CONTRIBUTING.md

```diff
@@ -1,17 +1,35 @@
-# Code of Conduct
+# Contributing to Balrog
 
-See [CODE_OF_CONDUCT](https://github.com/mozilla-releng/balrog/blob/master/CODE_OF_CONDUCT.md)
+Thank you for your interest in contributing to Balrog! To effectively work on this project, you will need to set up your development environment and familiarize yourself with our contribution guidelines.
 
-# Contributing to Balrog
+## Development Environment Requirements
+
+Before you start contributing, ensure that you have the following tools and dependencies:
+
+- Docker
+- Docker Compose
+- Tox
+
+For detailed instructions on setting up your development environment, please refer to the [README](https://github.com/mozilla-releng/balrog/blob/master/README.rst) file. It contains comprehensive information on installation and configuration.
+
+## Finding Issues to Work On
+
+If you're new to Balrog, we recommend starting with a "first bug." You can find first bug issues by visiting the [First Bug](https://github.com/mozilla-releng/balrog/issues?q=is%3Aissue+is%3Aopen+label%3Afirst-bug) label. These issues are suitable for newcomers and serve as a great introduction to the project.
+
+For more experienced contributors, you can explore issues with the following labels:
 
-In order to effectively work on Balrog you will need to be able to both run its tests and local development environment. You will need a recent version of Docker, Docker Compose, and tox to do these things.
+- [Admin](https://github.com/mozilla-releng/balrog/labels/admin): Administrative tasks
+- [Agent](https://github.com/mozilla-releng/balrog/labels/agent): Agent-related tasks
+- [Public](https://github.com/mozilla-releng/balrog/labels/public): Public-facing tasks
```

Comment:
`Admin`, `Agent`, and `Public` are all different applications written in Python. (`admin` and `public` are webapps, `agent` is a small damon.)